### DISCUSSION TITLE
Keldagrim Tasks

### DISCUSSION THREAD

**Discussion by K-X-S** [runelite/runelite#17425] [Ideas]
I'm sure this may get dismissed as this only affects a niche number of accounts...
But any accs wanting to keep 10hp/low hp and build things like 10hp def/ atk can utlize Keldagrim tasks for a good chunk more xp /hr than digging for fossils in the museum..
However, this seems to be a random 1/7 chance to which dwarf at the time is offering a task. There's only 2 that offer atk/def xp respectively. You spend most of the time running back and forth between both sides of Keldagrim talking to each in hopes you can ignore the 5 others and that the 1 of 2 you may selectively want, offers. 

Curious if we could get an indicator like Arceuus Library where were told who is currently offering a task so we can talk to them, ignore them and so on until the desired one is offering?

**Comment by TankyHealth** [comment]
This, please!
